### DISCUSSION TITLE
PrismaClient is unable to be run in the browser [nextJS error]

### DISCUSSION THREAD

**Discussion by knnedy** [prisma/prisma#16152] [Q&A] [answered]
### Bug description

Intialized prisma on config.db and exported the single instance of prisma in to the api folder and reproduced the error

### Expected behavior

To get items from the database

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 
Node v16.15.1

### Prisma Version

```
Prisma  v4.3.1
```

**Comment by nurul3101** [comment, marked as answer]
Hello @IconicDexter 👋 
It seems you are trying to run Prisma in the frontend which isn't supported.
Here's a related [GitHub Issue](https://github.com/prisma/prisma/issues/6219) which discusses the same.
Can you confirm if you are using Prisma in [NextJS API routes](https://nextjs.org/docs/api-routes/introduction) or in [getServerSideProps/getStaticProps](https://nextjs.org/docs/basic-features/data-fetching/overview#getserversideprops-server-side-rendering) methods?

**Reply by vladi-stevanovic** [reply]
👋 Hi @IconicDexter , did you perhaps have a chance to check Nurul's recommendation? Let us know if you still have any questions! 

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
